### ISSUE TITLE
Boot videos become laggy when "Use as Wake Movie" is selected in Steam Settings

### ISSUE THREAD

**Issue by ParkerPattillo** [TheLogicMaster/SDH-AnimationChanger#59]
I'm trying to mainly have boot videos as something that play when I take the deck out of startup since I don't reboot my deck that often. Steam has a setting to set the boot video under deck_startup.webm as a Wake Movie. Unfortunately no matter what video I have selected in Animation Changer the boot video becomes laggy either at the beginning or all the way throughout on wake up. 

Wondering if there was a fix to this in Animation Changer or if the plugin could have the capability in the future to set the videos to play as a Wake Movie without enabling it in the steam settings. On that same note it would be cool to see the Shuffle feature work the same for Wake Movies as it does for boot videos.

Thank you!

**Comment by TheLogicMaster** [comment]
So, all the plugin does is symlink downloaded animations into ~/.steam/root/config/uioverrides/movies. deck_startup.webm is for the boot animation, deck-suspend-animation.webm is for going to sleep when not in game, and deck-suspend-animation-from-throbber.webm is for going to sleep when in game. Since the settings customization menu was added, deck_startup.webm now has to be selected in said menu for the plugin to function, otherwise only the suspend animations should take effect.

**Comment by ParkerPattillo** [comment]
Yes deck_startup.webm is selected in that Steam menu. My problem is that in that same Customization menu the option "Set as Wake Movie" causes my downloaded animations to become laggy after wakeup. When that option is not selected and the animations play on boot there's no lag at all, the plugin works as intended but like I said I don't reboot the deck terribly often. 

My question I guess is more along the lines of why that lag on the boot animations is occurring when that "Wake Movie" setting is selected and if there is anyway to fix it or is that more of a Steam OS interfering kind of a thing.

I feel like I'm doing a bad job of explaining this but if you go on your personal deck and enable the same "Set as Wake Movie" option in the Customization menu I'm sure that the same lag will occur to whatever your boot animation is set to in the plugin.

**Comment by ParkerPattillo** [comment]
Yes deck_startup.webm is selected in that Steam menu. My problem is that in that same Customization menu the option "Set as Wake Movie" causes my downloaded animations to become laggy after wakeup. When that option is not selected and the animations play on boot there's no lag at all, the plugin works as intended but like I said I don't reboot the deck terribly often.

My question I guess is more along the lines of why that lag on the boot animations is occurring when that "Wake Movie" setting is selected and if there is anyway to fix it or is that more of a Steam OS interfering kind of a thing.

I feel like I'm doing a bad job of explaining this but if you go on your personal deck and enable the same "Set as Wake Movie" option in the Customization menu I'm sure that the same lag will occur to whatever your boot animation is set to in the plugin. It's inconsistent at best, at worse it just freezes then speeds up really quickly to catch up with the audio.

**Comment by ghost** [comment]
Hey @ParkerPattillo the same exact thing happens for me. And it's very inconsistent. Sometimes it plays better than others, i've messed with my clock speeds and wattage with the same results so I don't think it's a performance issue. I really don't know what it could be.

**Comment by RANGER592** [comment]
I am also having this issue still.

**Comment by KirbyFighter** [comment]
Did anybody figure out what is causing this issue?
